service/server.py

```diff
@@ -55,26 +56,49 @@ def healthcheck():
 
 
 def get_property_address(postcode):
-    client = Elasticsearch([ELASTIC_SEARCH_ENDPOINT])
-    search = Search(
-        using=client, index='landregistry', doc_type='property_by_postcode_2')
+    search = create_search('property_by_postcode_2')
```

Comment:
It's good. At some point in the future we should move the doc type names into the config, so that the code doesn't change when we switch from one to another.